build-image/first-boot.bash

```diff
@@ -27,7 +31,7 @@ source /etc/openhabian.conf
 echo "OK"
 
 echo -n "$(timestamp) [openHABian] Starting webserver with installation log... "
-if hash python 2>/dev/null; then
+if hash python3 2>/dev/null; then
```

Comment:
python3 again... what is the reason to change.
for my current production box, default still points to 2.7, though both 2 and 3 are installed.

Reply:
The webserver for the webif to show install status (boot.log) didn't work in my testing.
Now if only I could remember all the details 9 months after ... but the fix was to move to Python3. As I didn't want to upgrade to P3 I just installed and explicitly called it.

Reply:
should not harm, but your explanation about webif problem with python2 is worth a comment in the code....
For mr ok to keep as py27 will be removed at some point in time anyway...

Reply:
ok, added a comment

So we're done ?
Merge ? I'll be at home tomorrow 😅